src/components/Tooltip/__stories__/TooltipContent.jsx

```diff
@@ -2,7 +2,7 @@ import React from "react";
 import "./TooltipContent.scss";
 
 const TooltipContent = () => {
-  return <div>Tooltip Content</div>;
+  return <div>Text</div>;
```

Comment:
Why we do this change?

Reply:
This component is only used inside Storybook, and in the new design, it contains "Text".